### DISCUSSION TITLE
[IntelliSense] Detect Duplicate CSS Styles Applied With Transition and Ease Classes

### DISCUSSION THREAD

**Discussion by manavm1990** [tailwindlabs/tailwindcss#8804] [Ideas]
As per the [documentation on `.transition`](https://tailwindcss.com/docs/transition-property), the CSS includes: `transition-timing-function: cubic-bezier(0.4, 0, 0.2, 1);`

As per [documentation on `.ease-in-out`](https://tailwindcss.com/docs/transition-timing-function#controlling-the-easing-curve), same is included: `transition-timing-function: cubic-bezier(0.4, 0, 0.2, 1);`

IntelliSense does a great job of alerting us whenever we use multiple Tailwind classes that apply the same rules.

However, we don't get this alert whenever we do: `transition ease-in-out`. These two classes duplicate the same CSS: `transition-timing-function: cubic-bezier(0.4, 0, 0.2, 1);`

The plugin doesn't pick this 1️⃣ up for some reason. 

No biggie, but instead of: ``transition ease-in-out`, same is done with: `transition`

**Comment by bradlc** [comment]
Hey @manavm1990. It looks like you may have confused `prettier-plugin-tailwindcss` for [Tailwind CSS IntelliSense](https://github.com/tailwindlabs/tailwindcss-intellisense), which is the VS Code extension that provides the warnings that you are describing. I'm going to move this issue to our discussions area as this is more of a feature request than a bug report. Please let me know if I've misinterpreted though!
